### PR TITLE
Load hero content from CMS

### PR DESCRIPTION
**Pull request by esirK**
## Description

Load hero content from CMS

Finishes https://github.com/CodeForAfrica/ui/issues/212

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
N/A
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/codeforafrica/content/pages/privacy-policy.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 title: PRIVACY
 subtitle: Code for Africa privacy policy
```
Comment (kilemensi) [inline apps/codeforafrica/content/pages/privacy-policy.md:3]:
```suggestion
```

File: apps/codeforafrica/src/lib/index.js [hunk 2]
```diff
@@ -171,7 +171,7 @@ async function getHomePageStaticProps() {
       title: "Code for Africa",
       sections: [
         {
-          ...getHero(),
+          ...getHero("index"),
```
Comment (kilemensi) [inline apps/codeforafrica/src/lib/index.js:174]: [`index`](https://github.com/CodeForAfrica/ui/blob/30e0fc31be6265fe87152bbbb3798188e5b00229/apps/codeforafrica/src/lib/api.netlify-cms/sections/getHero.js#L9) is the default though

**Review by kilemensi — APPROVED** [review]
🚀

File: apps/codeforafrica/src/lib/index.js (hunk 2 above)
Comment (esirK) [reply apps/codeforafrica/src/lib/index.js:174]: yeah. Just being more explicit. Maybe remove the default?

Comment (kilemensi) [reply apps/codeforafrica/src/lib/index.js:174]: You're the one who came up with the default if I remember correctly.

Comment (esirK) [reply apps/codeforafrica/src/lib/index.js:174]: yeah @kilemensi . Just rethinking the design a little bit.

Comment (kilemensi) [reply apps/codeforafrica/src/lib/index.js:174]: Sure Mr. API Design.

**Pull request merged**